io.github.gitahead.Gitahead.json

```diff
@@ -0,0 +1,130 @@
+    {
+      "name": "libsectre"
+    },
+    "shared-modules/intltool/intltool-0.51.json",
+    {
+      "name": "intltool"
+    },
+    {
+      "name": "git",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 $(which git) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-clang-format) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-cvsserver) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-receive-pack) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-shell) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-upload-archive) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-upload-pack) ${FLATPAK_DEST}/bin/"
+      ]
+    },
+    {
+      "name": "graphviz",
+      "cleanup": [
+        "/share/man",
+        "/share/graphviz/doc"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.com/graphviz/graphviz/-/archive/2.47.0/graphviz-2.47.0.tar.gz",
+          "sha256": "4b7f61c7c04f39b67b2c87977633d61d9f72ee6f93ff4376acdce140128431d4"
+        }
+      ]
+    },
+    {
+      "name": "libgnome-keyring",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--enable-vala=no",
+        "--enable-introspection=no"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download-fallback.gnome.org/sources/libgnome-keyring/3.9/libgnome-keyring-3.9.1.tar.xz",
+          "sha256": "8eaca3d4f04ba834acb6c5037b26c3f0837ee4b71ccb380e7810052f7cde381b"
+        },
+        {
+          "type": "file",
+          "path": "gitahead.patch"
+        }
+      ],
+      "cleanup":[
+        "/share/gtk-doc"
+      ]
+    },
+    {
+      "name": "gitahead",
+      "buildsystem": "simple",
+      "cleanup": [
+        "/usr/local/share/",
+        "/usr/local/man",
+        "/usr/local/share/doc",
+        "/usr/local/share/man",
+        "/usr/local/Resources/*.html",
+        "/share/man"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/gitahead/gitahead.git",
+          "commit":"bb747caef098140d9674990b1d79d096d1eac8df"
+        },
+        {
+          "type": "patch",
+          "path": "gitahead.patch"
+        },
+        {
+          "type": "file",
+          "path": "io.github.gitahead.Gitahead.xml"
+        },
+        {
+          "type": "file",
+          "path": "io.github.gitahead.Gitahead.desktop"
+        },
+        {
+          "type": "script",
+          "dest-filename": "run.sh",
+          "commands": [
+            "/app/usr/local/GitAhead"
+          ]
+        }
+      ],
+      "build-commands": [
+        "mkdir -p build/release",
+        "cd build/release && cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_LINK_DIRECTORIES=${FLATPAK_DEST} -DCMAKE_PREFIX_PATH=/usr/lib ../.. && ninja &&  DESTDIR=${FLATPAK_DEST} ninja install",
```

Comment:
Since this is cmake, `"buildystem": "cmake-ninja"` should work.

Reply:
I did test that it doesn't work because of git ahead using a really weird build layout

Reply:
Using `/usr/lib` for prefix is just wrong though. It should be `/app`.

Reply:
Also it works fine here if I set `builddir` to `true`.